.changeset/real-carpets-press.md

```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/myaccount": patch
+"@wso2is/core": patch
+"@wso2is/i18n": patch
```

Comment:
Since this is a new feature, let's bump myaccount to minor.

```suggestion
"@wso2is/myaccount": minor
"@wso2is/core": patch
"@wso2is/i18n": patch
```

Reply:
Changed